python/BioSimSpace/FreeEnergy/_relative.py

```diff
@@ -696,6 +1278,159 @@ def _difference(self, pmf_ref):
         # Now call the staticmethod passing in both PMFs.
         return Relative.difference(pmf, pmf_ref)
 
+    @staticmethod
+    def check_overlap(overlap, estimator="MBAR"):
+        """Check the overlap of an FEP leg. 
+
+           Parameters
+           ----------
+
+           overlap : [ [ float, float, ... ] ], numpy.matrix
+               The overlap matrix. This gives the overlap between lambda windows.
+
+           estimator : str
+               Must be "MBAR" for checking the overlap matrix.
+
+           Returns
+           -------
+
+           Message - gives a warning if the overlap is less than 0.03 for any off-diagonal.
```

Comment:
Can this return nothing and throw an exception, or return a boolean, e.g. with False being poor overlap. I imagine a workflow where the user with have call `checkOverlap` in a script and decide whether to proceed (or submit more jobs) based on the result.